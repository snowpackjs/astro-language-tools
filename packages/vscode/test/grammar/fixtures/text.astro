text<div>text</div>
<div>text</div>text
text{"text"}
{"text"}text