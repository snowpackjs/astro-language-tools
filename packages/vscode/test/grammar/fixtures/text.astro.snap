>text<div>text</div>
#^^^^ source.astro text.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
#     ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#        ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#         ^^^^ source.astro text.astro
#             ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#               ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#                  ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
><div>text</div>text
#^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^ source.astro meta.scope.tag.div.astro meta.tag.start.astro entity.name.tag.astro
#    ^ source.astro meta.scope.tag.div.astro meta.tag.start.astro punctuation.definition.tag.end.astro
#     ^^^^ source.astro text.astro
#         ^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#           ^^^ source.astro meta.scope.tag.div.astro meta.tag.end.astro entity.name.tag.astro
#              ^ source.astro meta.scope.tag.div.astro meta.tag.end.astro punctuation.definition.tag.end.astro
#               ^^^^ source.astro text.astro
>text{"text"}
#^^^^ source.astro text.astro
#    ^ source.astro punctuation.section.embedded.begin.astro
#     ^^^^^^ source.astro meta.embedded.expression.astro source.tsx
#           ^ source.astro punctuation.section.embedded.end.astro
>{"text"}text
#^ source.astro punctuation.section.embedded.begin.astro
# ^^^^^^ source.astro meta.embedded.expression.astro source.tsx
#       ^ source.astro punctuation.section.embedded.end.astro
#        ^^^^ source.astro text.astro